c/tskit/tables.c

```diff
@@ -4530,12 +4530,12 @@ table_sorter_free(table_sorter_t *self)
  * msprime can then #include this and also use it. msprime needs a different
```

Comment:
We should delete this comment - it'll just be confusing now that the TODO is a DONE!